filebeat/docs/filebeat-options.asciidoc

```diff
@@ -1,544 +1,68 @@
-reached. After having backed off multiple times from checking the file, the wait time
-will never exceed `max_backoff` regardless of what is specified for  `backoff_factor`.
-Because it takes a maximum of 10s to read a new line, specifying 10s for `max_backoff` means that, at the worst, a new line could be added to the log file if Filebeat has
-backed off multiple times. The default is 10s.
-
-Requirement: max_backoff should always be set to `max_backoff <= scan_frequency`. In case `max_backoff` should be bigger, it is recommended to close the file handler instead let the prospector pick up the file again.
-
-[float]
-==== `backoff_factor`
-
-This option specifies how fast the waiting time is increased. The bigger the
-backoff factor, the faster the `max_backoff` value is reached. The backoff factor
-increments exponentially. The minimum value allowed is 1. If this value is set to 1,
-the backoff algorithm is disabled, and the `backoff` value is used for waiting for new
-lines. The `backoff` value will be multiplied each time with the `backoff_factor` until
-`max_backoff` is reached. The default is 2.
-
-[float]
-[[harvester-limit]]
-==== `harvester_limit`
-
-The `harvester_limit` option limits the number of harvesters that are started in parallel for one prospector. This directly relates
-to the maximum number of file handlers that are opened. The default for `harvester_limit` is 0, which means there is no limit. This
-configuration is useful if the number of files to be harvested exceeds the open file handler limit of the operating system.
-
-Setting a limit on the number of harvesters means that potentially not all files are opened in parallel. Therefore we recommended that you use
-this option in combination with the `close_*` options to make sure harvesters are stopped more often so that new files can be
-picked up.
-
-Currently if a new harvester can be started again, the harvester is picked randomly. This means it's possible that the harvester for a file that was just closed and then updated again might be started instead of the harvester for a file that hasn't been harvested for a longer period of time.
-
-This configuration option applies per prospector. You can use this option to indirectly set higher priorities on certain prospectors
-by assigning a higher limit of harvesters.
-
-[float]
-==== `enabled`
-
-The `enabled` option can be used with each prospector to define if a prospector is enabled or not. By default, enabled is set to true.
-
-[float]
-[[max-message-size]]
-==== `max_message_size`
-
-When used with `type: udp`, specifies the maximum size of the message received over UDP. The default is 10240.
-
-[float]
-[[config-containers]]
-==== `containers`
+You can configure {beatname_uc} to use the following inputs:
 
-experimental[]
+// REVIWERS: I would like to make the following list generated, if possible. Can
+// we reuse the logic that we use for generating the modules list?
```

Comment:
This would be awesome.

Reply:
Hmm...I looked at the logic that we use to create the list of modules for the docs, and I don't like that it recurses the directory where the source files are stored. I'd rather have something that looks at the hierarchy defined by the asciidoc markup (rather than relying on the source files to be in a particular directory). 

Our doc tools really should support a macro or something for generating "jump" lists or tables. This is not a new concept. I could do it in FrameMaker 15 years ago. Rather than coming up with a solution just for Beats, I'm going to open a request in the docs repo.

Reply:
I've created this issue to track the enhancement request: https://github.com/elastic/docs/issues/324